src/components/SpecialResolution/Resolution.vue

```diff
@@ -257,6 +260,10 @@ export default class Resolution extends Vue {
   async updateIsEditingSpecialResolution (val): Promise<void> {
     await this.setEditingSpecialResolution(val)
   }
+
+  onResolutionDate (date: string) {
+    this.changedResolutionDate = date
+  }
```

Comment:
This is fine but next time I'd be tempted to just put this code on line 133.

(In this case, one less function is one less thing to look at when trying to understand this code.)

Reply:
ok will do it next time, thanks!